Neos.Neos/Classes/FrontendRouting/NodeAddress.php

```diff
@@ -59,16 +50,10 @@ public function serializeForUri(): string
             . '__' . $this->nodeAggregateId->value;
     }
 
-    public function isInLiveWorkspace(): bool
-    {
-        return $this->workspaceName->isLive();
-    }
-
     public function __toString(): string
     {
         return sprintf(
-            'NodeAddress[contentStream=%s, dimensionSpacePoint=%s, nodeAggregateId=%s, workspaceName=%s]',
-            $this->contentStreamId->value,
+            'NodeAddress[dimensionSpacePoint=%s, nodeAggregateId=%s, workspaceName=%s]',
```

Comment:
I hope that we don' relay on this string representation (=> do we really need it, what about using [__debugInfo()](https://www.php.net/manual/en/language.oop5.magic.php#object.debuginfo) instead?) – and if we keep it: Why did you decide to move the workspace name to the end?

Reply:
we probably dont even need the `__toString` anymore at this time ... but i kept it ... it will all go away. Its legacy.